app/services/trading_strategies.py

```
@@ -4215,6 +4614,304 @@ async def _estimate_daily_volatility(self, symbol: str) -> float:
+                if current_ratio > 0.5:
+                    return "REDUCE_POSITIONS - High risk with excessive margin usage"
+                else:
+                    return "MONITOR - High risk market conditions"
+            elif risk_level == "MEDIUM":
+                if current_ratio > 0.7:
+                    return "CAUTION - Consider reducing margin usage"
+                else:
+                    return "NORMAL - Current margin usage acceptable"
+            else:
+                return "OPTIMAL - Low risk, efficient margin usage"
+        except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)